java/arcs/sdk/wasm/WasmJsInterop.kt

```diff
@@ -123,24 +121,24 @@ fun connectHandle(
     canWrite: Boolean
 ): WasmAddress {
     return particlePtr
-        .toObject<Particle>()
+        .toObject<WasmParticle>()
         ?.connectHandle(handleName.toKString(), canRead, canWrite)
         ?.toAddress() ?: 0
```

Comment:
Should the RHS of the elvis operator be an exception throw instead of returning 0?

Reply:
I'm not sure... I'm inclined to leave it as it is for now.